src/app/inputs/inputs.component.ts

```diff
@@ -1,44 +1,42 @@
-import { Component } from '@angular/core';
+import { Component, EventEmitter, Output } from '@angular/core';
 import { ApiService } from '../services/api.service';
-import { Task } from '../interface/task';
+import { Task } from '../interfaces/task.interface';
 
 @Component({
   selector: 'app-inputs',
   templateUrl: './inputs.component.html',
   styleUrls: ['./inputs.component.css']
 })
 export class InputsComponent {
-  
 
   constructor(private apiService: ApiService) { }
   
   addTask() {
-
     const inputTitle = document.getElementById('input-ul') as HTMLInputElement | null;
     const taskTitle = inputTitle?.value;
     const inputDesc = document.getElementById('desc-ul') as HTMLInputElement | null;
     const taskDesc = inputDesc?.value;
+
     if (!taskTitle || !taskDesc) {
-      return
+      return;
     }
+
     const newTask: Task = {
       title: taskTitle,
       description: taskDesc,
       done: false
     }
-    this.apiService.createTask(newTask)
+    this.apiService.createTask(newTask).then(() => location.reload())
```

Comment:
You should use a dynamic way to load new cards, as forcing a reload might take time for the user to load the web app again and disrupt their workflow.

Reply:
We did not have time to fix all of this calls, we might need a service refactor

Reply:
That's a good idea for the next sprint